compiler/src/dmd/backend/x86/disasm86.d

```diff
@@ -1757,14 +1757,16 @@ L2:
     }
 
     puts(p0);
-    put('\t');
-    if (waitflag)
+    put(' ');
+    if (0 && waitflag)
```

Comment:
Is there a use for this dead branch?

Reply:
Support for the older x87 instruction set.